src/epgsql_sock.erl

```diff
@@ -126,30 +126,48 @@ handle_cast(cancel, State = #state{backend = {Pid, Key},
                              gen_tcp -> inet:peername(TimedOutSock);
                              ssl -> ssl:peername(TimedOutSock)
                          end,
-    SockOpts = [{active, false}, {packet, raw}, binary],
+    SockOpts = [{active, false}, {packet, raw}, binary,
+                {keepalive, true}],
```

Comment:
Is the addition of the keepalive option necessary or is this just something we are adding since we happen to be touching this code anyway?